torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3084,7 +3084,7 @@ def sample_inputs_max_pool(op_info, device, dtype, requires_grad, **kwargs):
 
 def error_inputs_max_pool1d(op_info, device, **kwargs):
     # error inputs when pad is negative
-    x = torch.rand([0, 1, 49], dtype=torch.float32)
+    x = torch.rand([0, 1, 49], device=device, dtype=torch.float32)
```

Comment:
Good catch -- why does this call `rand` and not `make_tensor`?

Reply:
Updated.